tests/collections/asr/test_asr_classification_model.py

```diff
@@ -191,6 +191,8 @@ def test_EncDecClassificationDatasetConfig_for_AudioToSpeechLabelDataset(self):
             'sample_rate',
             'normalize_audio',
             'augmentor',
+            'bucketing_batch_size',
```

Comment:
These new parameters need to be added to the config file in the nemo/collections/asr/models/configs/classification_models_config.py.

Reply:
Also please add them to the config files in the example folder.

Reply:
Done